app/controllers/chargeback_controller.rb

```diff
@@ -728,6 +728,10 @@ def get_cis_all
       EmsCluster.all.each do |cl|
         @edit[:cb_assign][:cis][cl.id] = cl.name
       end
+    elsif @edit[:new][:cbshow_typ] == "tenants"
+      Tenant.all.each do |cl|
+        @edit[:cb_assign][:cis][cl.id] = cl.name
+      end
```

Comment:
Can you add a test for this method and change it to something like this?

```ruby
@edit[:cb_assign][:cis] = {}
klass =
  if @edit[:new][:cbshow_typ] == "enterprise"
    MiqEnterprise
  else
    @edit[:new][:cbshow_typ].classify.constantize
  end

klass.all.each do |instance|
  @edit[:cb_assign][:cis][instance.id] = instance.name
end
```

We can do this because MiqEnterprise only ever has a single row, and the name of that single row is "Enterprise" (what's hard coded in this method).

If we changed cbshow_typ to be consistent (miq_enterprise), then we can further
simplify this method.